frontend/src/views/LevelView.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <div class="Container">
+    <img src="@/assets/images/logo-and-lettering.svg" alt="Saci Logo" />
+    <h1>{{ newLevelTitle }}</h1>
+    <h2>Basic Information</h2>
+    <div class="NameContainer">
+      <h5>Name</h5>
+      <ElInput placeholder="Level Name" v-model="newLevel.name" />
+    </div>
+    <div class="LinkAndCoeficientsContainer">
+      <div class="CoeficientsContainer">
+        <h5>Coeficient Range</h5>
+        <span>
+          <ElInput placeholder="Min Range" v-model="newLevel.minCoeficient"></ElInput>
+          <h4>to</h4>
+          <ElInput placeholder="Max Range" v-model="newLevel.maxCoeficient"></ElInput>
+        </span>
+      </div>
+      <div class="LinkField">
+        <h5>Link to Knowledge Path</h5>
+        <ElInput placeholder="Link" v-model="newLevel.link"></ElInput>
+      </div>
+    </div>
+    <div class="SaveButton">
+      <ElButton type="primary" @click="createLevel()"> Save </ElButton>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { Role } from '@/domain/Role'
+import type { Level } from '@/domain/Level'
+import { useRolesStore } from '@/stores/roles'
+import { ref, onMounted, computed } from 'vue'
+import { ElInput, ElButton } from 'element-plus'
+import { ErrorMessage, SuccessMessage } from '@/services/messages'
+import { useRouter } from 'vue-router'
+import { HttpServerError } from '@/services/http'
+
+const roleStore = useRolesStore()
+const role = ref<Role>()
+const newLevel = ref<Level>({ name: '', minCoeficient: 0, maxCoeficient: 0, link: '' })
+const router = useRouter()
+const newLevelTitle = computed(() => {
+  return `${role.value?.name || ''} - ${newLevel.value.name || 'New Level'}`
+})
+
+onMounted(async () => {
+  role.value = await roleStore.getRoleById('1')
+})
+
+const redirectToRoleView = () => {
+  router.push('/role-view')
+  SuccessMessage('Your level was created')
+}
+
+const createLevel = async () => {
+  if (
+    newLevel.value.name == '' ||
+    newLevel.value.minCoeficient == 0 ||
```

Comment:
This validation would pass if the level had a name, a link, but the coefficients be empty, instead of zero. Right?

Besides that, it's possible for the minCoefficient to be 0

To avoid this, put the validation of maxCoefficient > 0, and put the input type of the field as number. Check this https://element-plus.org/en-US/component/input-number.html